charmhelpers/contrib/openstack/audits/__init__.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2019 Canonical Limited.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""OpenStack Security Audit code"""
+
+import apt_pkg
+#  noqa below is because we import apt for a side-effect
+#  initializing apt_pkg.version_compare's _system
+import apt # noqa: F401,E261
+import collections
+from enum import Enum
+import traceback
+
+from charmhelpers.fetch.ubuntu import get_upstream_version
+
+from charmhelpers.core.hookenv import (
+    action_fail,
+    action_set,
+)
```

Comment:
I tend to prefer the style of

```python
import charmhelpers.core.hookenv as hookenv

# and then use it as

hookenv.action_fail(...)
```

This is just to signal in code where something came from.  However, I'm not overly hung up on this as the rest of the code in charmhelpers tends to be of this style.